### PR TITLE
Add variables for services configuration patches

### PR DESCRIPTION
**Pull request by eshulman2**
Added variables for services configuration patches to allow overridding them and use different configuration

### CODE REVIEW

File: tests/roles/ironic_adoption/defaultes/main.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                    metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+                spec:
+                  type: LoadBalancer
+        ironicConductors:
+        - replicas: 1
+          networkAttachments:
+          - baremetal
+          provisionNetwork: baremetal
+          storageRequest: 10G
+          storageClass: {{ storage_class_name }}
```
Comment (jistr) [inline tests/roles/ironic_adoption/defaultes/main.yaml:27]: We may need to add a default for storage_class_name too, to make sure it is already defined when the ironic_patch value is being processed. I think when the template was in `tasks.yaml` it relied on the variable being included from OVN role defaults, but that may not be a reliable way anymore. We should probably copy the default from the OVN file into this file too. https://github.com/openstack-k8s-operators/data-plane-adoption/blob/eca663af5a80df64a6b78388ce5a7d9181615fa0/tests/roles/ovn_adoption/defaults/main.yaml#L2

**Comment by pinikomarov** [comment]
right , this looks good to me

File: tests/roles/ironic_adoption/defaultes/main.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
+---
```
Comment (jistr) [inline tests/roles/ironic_adoption/defaultes/main.yaml:1]: Just noticed a typo: the directory is called "defaultes". Ironic adoption is not tested in CI so if you want to have CI feedback, you can let the current CI jobs finish. We should just fix this before merging.

Comment (eshulman2) [reply tests/roles/ironic_adoption/defaultes/main.yaml:1]: Completely missed it, thanks will fix after CI

Comment (eshulman2) [reply tests/roles/ironic_adoption/defaultes/main.yaml]: fixed this and the neutron typo that failed the gate

**Review by jistr — APPROVED** [review]

**Pull request merged**